web/concrete/src/User/User.php

```diff
@@ -125,10 +127,10 @@ public function getUserInfoObject()
 
     public function __construct()
     {
-        $app = Application::getFacadeApplication();
+        $this->app = Application::getFacadeApplication();
```

Comment:
I made this a class property, we already instantiate it in the constructor, with this we can avoid further calls to getFacadeApplication